rivers/books_drupal.json

```diff
@@ -0,0 +1,131 @@
+{
+  "type" : "remote",
+  "remote" : {
+    "urlGetDocuments"               : "http://developer-drupal.web.stage.ext.phx2.redhat.com/drupal/books",
+    "listDocumentsMode"             : "simple",
+    "spacesIndexed"                 : "MAIN",
+    "timeout"                       : "20s",
+    "indexUpdatePeriod"             : "10m",
+    "maxIndexingThreads"            : 1
+  },
+  "index" : {
+    "index"                         : "data_jbossdeveloper_book",
+    "type"                          : "jbossdeveloper_book",
+    "remote_field_document_id"      : "nid",
+    "field_document_id"             : "sys_id",
+    "fields": {
+      "authors": {"remote_field": "prep_sys_authors"},
+      "average_rating": {"remote_field": "field_average_rating"},
+      "categories": {"remote_field": "field_categories"},
+      "info_link": {"remote_field": "field_preview_url"},
+      "isbn": {"remote_field": "prep_isbn"},
+      "preview_link": {"remote_field": "field_preview_url"},
+      "published_date": {"remote_field": "test_sys_created_date"},
```

Comment:
This field seems redundant to sys_created, plus 'test_sys_created_date' doesn't exist.